docs/jupyter_book_notes.md

```diff
@@ -0,0 +1,77 @@
+<!-- :orphan: is needed so we don't get a TOC warning -->
+```{eval-rst}
+:orphan:
+```
```

Comment:
Is this file development only? Intended to be merged?

Reply:
Yeah, development only as I was playing with Jupyter Book. I'll slink them away [elsewhere](https://ajfriend.github.io/pydeck-h3/jupyter_book_notes.html) for now :)